clients/java/src/main/java/io/camunda/zeebe/client/impl/http/ApiResponseConsumer.java

```diff
@@ -43,7 +43,7 @@ final class ApiResponseConsumer<T>
   @Override
   protected ApiResponse<T> buildResult(
       final HttpResponse response, final ApiEntity<T> entity, final ContentType contentType) {
-    return new ApiResponse<>(response.getCode(), entity);
+    return new ApiResponse<>(response.getCode(), response.getReasonPhrase(), entity);
```

Comment:
🔧 Those changes were a fix, please leave them in 👍

Reply:
This change is not in the stable branch then 👀

Reply:
❌

Reply:
For reference: those changes are needed when communicating with an 8.6 cluster to appropriately receive error messages. We need those changes in the 8.6 client in Zeebe and Camunda classes. This is currently not backported to 8.5 since it's meant to be compatible with 8.5 clusters, if I am not mistaken.